src/logic/DataUploader.ts

```diff
@@ -0,0 +1,31 @@
+const MINUTE = 1000 * 60;
+const INTERVAL_DURATION = 1 * MINUTE;
```

Comment:
Currently using `DataConfig.ts` under `constants` for storing config. Do you think it's better to move it there?

Reply:
yes it is good idea!